docs/docs/mdx/programmatically-creating-pages.md

```diff
@@ -32,7 +32,7 @@ root.
 
 > **NOTE**: `gatsby-mdx` uses `.mdx` by default as a file extension to
 > recognize which files to use. You can also [use `.md` as a file
-> extension](api-reference/options/extensions) if you want.
+> extension](https://gatsby-mdx.netlify.com/api-reference/options/extensions) if you want.
```

Comment:
Should this API reference stuff move to the Gatsby docs?

Reply:
Should we do those in a separate PR?

Reply:
Good idea, they can certainly happen incrementally, there will probably be a decent amount of work that goes into that.